providers/aws/services/vpc/vpc_endpoint_services_allowed_principals_trust_boundaries/vpc_endpoint_services_allowed_principals_trust_boundaries.metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "Provider": "aws",
+  "CheckID": "vpc_endpoint_services_allowed_principals_trust_boundaries",
+  "CheckTitle": "Find trust boundaries in VPC endpoint services allowlisted principles.",
+  "CheckType": ["Infrastructure Security"],
+  "ServiceName": "vpc",
+  "SubServiceName": "service_endpoint",
+  "ResourceIdTemplate": "arn:partition:service:region:account-id:resource-id",
+  "Severity": "medium",
+  "ResourceType": "AwsEc2Vpc",
+  "Description": "Find trust boundaries in VPC endpoint services allowlisted principles.",
+  "Risk": "Account VPC could be linked to other accounts.",
+  "RelatedUrl": "",
+  "Remediation": {
+    "Code": {
+      "CLI": "",
+      "NativeIaC": "",
+      "Other": "https://docs.bridgecrew.io/docs/networking_9#aws-vpc-endpoints-are-exposed",
```

Comment:
Check it again with the CloudMatos remediation

Reply:
No remediation in ClooudMatos.